src/utils.ts

```diff
@@ -425,13 +425,19 @@ export const tagBackportReviewers = async ({
   }
 
   if (Math.max(reviewers.length, teamReviewers.length) > 0) {
-    await context.octokit.pulls.requestReviewers(
-      context.repo({
-        pull_number: targetPrNumber,
-        reviewers,
-        team_reviewers: teamReviewers,
-      }),
-    );
+    try {
+      await context.octokit.pulls.requestReviewers(
+        context.repo({
+          pull_number: targetPrNumber,
+          reviewers,
+          team_reviewers: teamReviewers,
+        }),
+      );
+    } catch (error) {
+      console.error(
+        `Failed to request reviewers for PR #${targetPrNumber}: ${error}`,
+      );
```

Comment:
```suggestion
      log(
        'tagBackportReviewers',
        LogLevel.ERROR,
        `Failed to request reviewers for PR #${targetPrNumber}`,
        error,
      );
```

There's a dedicated log util which is used elsewhere in this file, so let's stay consistent and use it here as well.